### PR TITLE
Refresh controls and bump version to v2.0-20221012

### PR DESCRIPTION
**Pull request by acskurucz**
- Create new version of controls based on working group discussions.
- Expose older versions on the website

### CODE REVIEW

File: packages/mvsp/mvsp.en.asciidoc [hunk 1]
```diff
@@ -131,6 +121,9 @@ h| Description
 | 3.4 Time to fix vulnerabilities
 | Produce and deploy patches to address application vulnerabilities that materially impact security within 90 days of discovery.
 
+| 3.5 Build process
+| Build processes must be fully scripted/automated and generate provenance (https://slsa.dev/spec/v0.1/levels[SLSA Level 1]).
```
Comment (ChrisJohnRiley) [inline packages/mvsp/mvsp.en.asciidoc:125]:
```suggestion
| Build processes must be fully scripted/automated and generate provenance (https://slsa.dev/spec/v0.1/levels[SLSA Level 1])
```

File: src/faq.en.asciidoc [hunk 2]
```diff
@@ -101,12 +105,12 @@ Implementation of strong encryption, and enforcement using Strict-Transport-Secu
...
 To ensure that cookies are also only transmitted via encrypted and integrity protected channels, cookies must use the Secure flag.
 
-| 2.3 Content Security Policy
+| 2.3 Security Headers
 | **What this control measures:** Appropriate browser protections are in place within your product and/or service to protect against common web threats.
 
 ---
 
-**Why this control is important:** Content Security Policy is a mechanism designed to make applications more secure against common web vulnerabilities, particularly cross-site scripting. By adopting a strict policy, which prevents the loading of untrusted scripts or plugins, an application can add an important defense-in-depth layer against markup injection attacks.
+**Why this control is important:** Content Security Policy is a mechanism designed to make applications more secure against common web vulnerabilities, particularly cross-site scripting while iframing websites could lead to unexpected vulnerabilities. By adopting strict policies, which prevents the loading of untrusted scripts or plugins and limiting being iframed an application can add important defense-in-depth layers against markup injection attacks.
```
Comment (ChrisJohnRiley) [inline src/faq.en.asciidoc:113]:
```suggestion
**Why this control is important:** Content Security Policy is a mechanism designed to make applications more secure against common web vulnerabilities, particularly cross-site scripting while iframing websites could lead to unexpected vulnerabilities. By adopting strict policies, which prevents the loading of untrusted scripts or plugins and limiting being iframed an application can add important defense-in-depth layers against markup injection attacks.

Security headers can also assist in providing isolation and protection against common attacks beyond those discussed above. You can find more information at [securityheaders.com](https://securityheaders.com).
```

File: src/faq.en.asciidoc [hunk 3]
```diff
@@ -48,7 +48,11 @@ Although more tightly scoped release testing may take place throughout the year,
 Using an independent company for annual third party testing is important as they approach the review without any pre-existing opinions on how the application should function. This greenfield viewpoint ensures no assumptions are made regarding the security of specific functions or areas of the service which could lead to a lack of coverage or depth of review.
 
 | 1.5 Training
-| **What this control measures:** Processes are in place to provide regular and ongoing security awareness training.
+| **What this control measures:** Processes are in place to provide regular and ongoing security awareness training. This training should include:
```
Comment (ChrisJohnRiley) [inline src/faq.en.asciidoc:51]:
```suggestion
| **What this control measures:** Processes are in place to provide regular and ongoing security awareness training. Example areas of training that should be considered are:
```

**Review by ChrisJohnRiley — COMMENTED** [review]
Added comments and recommendations where appropriate.

I've added some recommended further reading points to a few areas where I think it makes sense.

File: src/faq.en.asciidoc [hunk 4]
```diff
@@ -67,9 +71,9 @@ Using an independent company for annual third party testing is important as they
 ---
 
 **Why this control is important:** Clear and timely notification of incidents is important to ensure customers are able to understand how an incident may affect their data, and prepare for further actions required to mitigate the effects of a breach. The initial notification is expected within 72 hours of discovery, however it should be augmented and extended upon once a more detailed analysis has occurred and root causes are identified. Not all information will be available within 72 hours, however it is important to ensure processes are in place to provide initial information within this timeframe.
-  
-| 1.8 Data sanitization
-| **What this control measures:** Data stored on removable or decommissioned hardware is appropriately handled.
+
+| 1.8 Data handling
+| **What this control measures:** Data is appropriately handled based on classification including retention on removable or decommissioned hardware.
```
[the remark this answers is not in the document]
Comment (acskurucz) [reply src/faq.en.asciidoc:76]: There was a discussion about adding "Maintain documented data classification and retention policies" and I adjusted the FAQ with that in mind. I revert the sentence.

File: src/faq.en.asciidoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (acskurucz) [reply src/faq.en.asciidoc]: Accepted the suggestion, thanks!

File: packages/mvsp/mvsp.en.asciidoc [hunk 5]
```diff
@@ -32,23 +32,23 @@ h| Description
 | 1.4 External testing
 | Contract a security vendor to perform annual, comprehensive penetration tests on your systems
 
-
 | 1.5 Training
-| Implement role-specific security training for your personnel that is relevant to their business function
+| Implement role-specific security training for your personnel that is relevant to their business function.
```
[the remark this answers is not in the document]
Comment (acskurucz) [reply packages/mvsp/mvsp.en.asciidoc]: Thanks, accepted those!

**Review by ChrisJohnRiley — APPROVED** [review]
Thanks for bringing together all the changes here. Looks good.

**Review by touzoku — APPROVED** [review]

**Pull request merged**